eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelTypes.scala

```diff
@@ -97,15 +97,7 @@ case class INPUT_RESTORED(data: HasCommitments)
 
 sealed trait BitcoinEvent extends PossiblyHarmful
```

Comment:
I believe the remaining `BitcoinEvent`s should be moved to the `Channel` companion object and made private.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1759/commits/8f11a4e2c4eb168af5ccad6f99ee985702c2129d